### DISCUSSION TITLE
Active Pen Compatibility

### DISCUSSION THREAD

**Discussion by BabyBlue2194** [saber-notes/saber#916] [Q&A] [answered]
Hello! 
I love this app! However, my Lazarite pen doesn't seem to register, and the command button seem to not work properly. This is the first app that has not worked for my pen. There is no pressure sensitivity, and the commands are not what I have set. Any ideas? I've tried opening up my pen settings, but nothing has changed.

**Comment by adil192** [comment, marked as answer]
Duplicate of https://github.com/adil192/saber/issues/671
